packages/frontend/src/hooks/useSavedDashboardFiltersOverrides.tsx

```diff
@@ -84,9 +96,14 @@ export const useSavedDashboardFiltersOverrides = () => {
         dispatch({ type: REMOVE_SAVED_FILTER_OVERRIDE, payload: item });
     };
 
+    const resetSavedFilterOverrides = () => {
+        dispatch({ type: RESET_SAVED_FILTER_OVERRIDES, payload: null });
```

Comment:
do we need `payload: null`

Reply:
It avoids some more convoluted type handling. The action wants a payload. This seems simple and explicit but if you have a pattern you prefer, I can change it!

Reply:
understood!